src/pages/docs/tutorials/tutorial-kitten-browser/data-and-state/index.md

```diff
@@ -14,58 +14,58 @@ order: 5
   * explain how spottable works (onMouseOver, onMouseLeave, onClick, onKeyPress, mouse emulation)
 -->
 
-In the [last step](../panels/), we focused on the app structure with Panels. Next, we'll investigate how custom and native events are used to notify your app of user actions, how state is defined for the app, and how to use events to update the state of your app.  We'll wrap up with a brief discussion on how to manage state data.
+In the [previous step](../panels/), we focused on the app structure with Panels. Next, we'll investigate how custom and native events are used to notify your app of user actions, how state is defined for the app, and how to use events to update the state of your app. We'll wrap up with a brief discussion on how to manage state data.
 
 ## Events in React
 
-[Native events in React](https://reactjs.org/docs/events.html) are registered using the camelCase version (`onMouseDown`) of their native name (`onmousedown`). If you wish to be notified of an event, you can pass a function as the value for the appropriate prop. The function will receive a synthetic event as its first argument, which is a cross-browser wrapper around the original event.
+[Native events in React](https://react.dev/reference/react-dom/components/common) are registered using the camelCase version (`onMouseDown`) of their native name (`onmousedown`). If you wish to be notified of an event, you can pass a function as the value for the appropriate prop. The function will receive a synthetic event as its first argument, which is a cross-browser wrapper around the original event.
```

Comment:
```suggestion
[Native events in React](https://react.dev/learn/responding-to-events) are registered using the camelCase version (`onMouseDown`) of their native name (`onmousedown`). If you wish to be notified of an event, you can pass a function as the value for the appropriate prop. The function will receive a synthetic event as its first argument, which is a cross-browser wrapper around the original event.
```

Reply:
In the context of following phrase, the link is to guide how to use events in React not to list up possible events.

Reply:
I accepted your suggestion.